docs/content/docs/deployment/azure-frs.md

```diff
@@ -33,7 +33,36 @@ Now that you have an instance of `FrsClient`, you can start using it to create o
 
 ### Token providers
 
-{{< placeholder >}}
+The [FrsAzFunctionTokenProvider]({{< relref "https://github.com/microsoft/FluidFramework/blob/main/experimental/framework/frs-client/src/FrsAzFunctionTokenProvider.ts" >}}) is an implementation of `ITokenProvider` which ensures your tenant key secret is not exposed in your client-side bundle code. The `FrsAzFunctionTokenProvider` takes in your Azure Function URL appended by `/api/GetFrsToken` along with the current user object. Later on, it makes an axios `GET` request call to your Azure function by passing in the tenantID, documentId and userID/userName as optional parameters.
+
+```javascript
+const config = {
+    tenantId: "myTenantId",
+    tokenProvider: new FrsAzFunctionTokenProvider("AZURE-FUNCTION-URL"+"/api/GetFrsToken", { userId:
+    "UserId", userName: "Test User"}),
+    orderer: "https://myOrdererUrl",
+    storage: "https://myStorageUrl",
+};
+
+const client = new FrsClient(config);
+```
+The user object can also hold optional additional user details such as the gender, address, email, etc.
+
+```javascript
+cont userDetails = {
+  email: "xyz@outlook.com",
+  address: "Redmond",
+};
+
```

Comment:
Let's also specify that the developer can add in any other fields here for this object that they would like to see returned in the `FrsMember` values from the audience in container services. I think a code sample showing the same additionalDetails field in the example above being parsed from audience would be helpful to show the entire loop